lib/msf/core/post/windows/dotnet.rb

```diff
@@ -0,0 +1,99 @@
+# -*- coding: binary -*-
+require 'msf/core/post/common'
+require 'msf/core/post/windows/registry'
+
+module Msf
+class Post
+module Windows
+
+#
+# This module serves to hold methods related to .NET framework
+#
+module Dotnet
+  include ::Msf::Post::Common
+  include ::Msf::Post::Windows::Registry
+
+  def initialize(info = {})
+    super
+    register_advanced_options(
```

Comment:
Sorry, I'm kind of coursed about these datastore options. Is there any code actually using these datastore options?